src/constants/specs.js

```diff
@@ -294,6 +298,7 @@ export const specs = {
   supervision: {
     icon: "star",
     draggable: true,
+    csv: true,
```

Comment:
As of now, we need `csv: true` in only `Book` and `Paper` so make sure it is not committed.